benchmarks/Microsoft.AspNetCore.SignalR.Microbenchmarks/DefaultHubDispatcherBenchmark.cs

```diff
@@ -0,0 +1,200 @@
+            private static readonly ChannelReader<int> ChannelReaderInstance = Channel.CreateUnbounded<int>();
+
+            public void Invocation()
+            {
+            }
+
+            public Task InvocationAsync()
+            {
+                return Task.CompletedTask;
+            }
+
+            public int InvocationReturnValue()
+            {
+                return 1;
+            }
+
+            public Task<int> InvocationReturnAsync()
+            {
+                return Task.FromResult(1);
+            }
+
+            public ValueTask<int> InvocationValueTaskAsync()
+            {
+                return new ValueTask<int>(1);
+            }
+
+            public IObservable<int> StreamObservable()
+            {
+                return ObservableInstance;
+            }
+
+            public Task<IObservable<int>> StreamObservableAsync()
+            {
+                return Task.FromResult(ObservableInstance);
+            }
+
+            public ValueTask<IObservable<int>> StreamObservableValueTaskAsync()
+            {
+                return new ValueTask<IObservable<int>>(ObservableInstance);
+            }
+
+            public ChannelReader<int> StreamChannelReader()
+            {
+                return ChannelReaderInstance;
+            }
+
+            public Task<ChannelReader<int>> StreamChannelReaderAsync()
+            {
+                return Task.FromResult(ChannelReaderInstance);
+            }
+
+            public ValueTask<ChannelReader<int>> StreamChannelReaderValueTaskAsync()
+            {
+                return new ValueTask<ChannelReader<int>>(ChannelReaderInstance);
+            }
+        }
+
+        [Benchmark]
+        public Task Invocation()
+        {
+            return _dispatcher.DispatchMessageAsync(_connectionContext, new InvocationMessage("123", "Invocation", null));
+        }
+
+        [Benchmark]
+        public Task InvocationAsync()
+        {
+            return _dispatcher.DispatchMessageAsync(_connectionContext, new InvocationMessage("123", "InvocationAsync", null));
+        }
+
+        [Benchmark]
+        public Task InvocationReturnValue()
+        {
+            return _dispatcher.DispatchMessageAsync(_connectionContext, new InvocationMessage("123", "InvocationReturnValue", null));
+        }
+
+        [Benchmark]
+        public Task InvocationReturnAsync()
+        {
+            return _dispatcher.DispatchMessageAsync(_connectionContext, new InvocationMessage("123", "InvocationReturnAsync", null));
+        }
+
+        [Benchmark]
+        public Task InvocationValueTaskAsync()
+        {
+            return _dispatcher.DispatchMessageAsync(_connectionContext, new InvocationMessage("123", "InvocationValueTaskAsync", null));
+        }
+
+        [Benchmark]
+        public Task StreamObservable()
+        {
+            return _dispatcher.DispatchMessageAsync(_connectionContext, new StreamInvocationMessage("123", "StreamObservable", null));
```

Comment:
This is exactly the same as `StreamChannelReader`

Reply:
No, that's a bug. Thanks